src/webaudio.js

```diff
@@ -305,7 +305,7 @@ export default class WebAudio extends util.Observer {
                 );
             }
             audio.autoplay = true;
-            var dest = this.ac.createMediaStreamDestination();
+            let dest = this.ac.createMediaStreamDestination();
```

Comment:
Hi, why `dist` and `audio`  were replaced with let instead of const, since immutable?

Reply:
because it was too much work to think about every var. I did with some but perhaps should've used let everywhere for consistency.

Reply:
but for core code it makes sense to be implicit, will change this one.

Reply:
> should've used let everywhere for consistency.

I think that where you're used const is logically correct, so can left as is